cli/commands/validate.mdx

```diff
@@ -19,7 +19,7 @@ flipt validate [flags]
 ### Behavior
 
 This command validates Flipt's declarative feature configuration files.
-It looks for features flag definitions in the same way as Flipts declarative backends.
+It looks for features flag definitions in the same way as Flipt's declarative backends.
 Checkout the documentation on [locating flag state](/configuration/storage#locating-flag-state) to learn more about this process.
 
```

Comment:
Maybe we should introduce Cue here by puting something here about how it uses Cue under the hood, and link to the Cue project?